vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -157,4 +157,4 @@ def test_page_build_left_side_removed(standard_px_chart):
     page = vm.Page(title="Single Page", components=[vm.Graph(id="scatter_chart", figure=standard_px_chart)])
     dashboard = vm.Dashboard(pages=[page])
     dashboard.pre_build()
-    assert "className='nav_control_panel'" not in str(page.build())
+    assert "id='left-main'" not in str(page.build())
```

Comment:
Ah, I think this is an outdated test. I've just checked and it doesn't make any sense anymore since we introduced the hidden attribute and moved the layout creation to the dashboard. 

I would kill this test completely, because this test was intended to check whether the left-main side collapses properly if there is only one page with no controls and no navigation. Even if I would re-write this to the dashboard.build(), "left-main" will actually be present in the component tree, but its attribute is hidden. So I think testing whether it then actually collapses need to be captured by the screenshot tests.

Reply:
In an ideal world I think we would test this as part of tests for `dashboard.build`. We'd make a page with no controls and no navigation and check for the hidden component in the build output using `assert_component_equal`. I wrote very similar tests for navigation to check that the nav panel is hidden when you have an accordion with only one item.

But writing those tests now is not needed, since it would take some time to do I know. I'd consider it part of the point I mention as future work writing more tests in #195.